src/himali/Assignment18/Program1.java

```diff
@@ -0,0 +1,38 @@
+package himali.Assignment18;
+
+import java.util.Scanner;
+
+public class Program1 {
+
+	void maxWordFrequency(String input) {
+		int max=0;
+		String maxOccurWord=null;
+		String[] inputArray=input.split(" ");
+		for(int index=0;index<inputArray.length;index++) {
+			String matchWord=inputArray[index];
+			int freqCount=0;
+			if(input.contains(matchWord)){
```

Comment:
this contains is not require as you are take matchWord from array, then definitely  that word will be in input string